src/operations/op_add.rs

```diff
@@ -0,0 +1,103 @@
+use crate::register::Register;
+
+use super::{sign_extend, update_flags};
+
+pub fn op_add(register: &mut Register, instr: u16) {
```

Comment:
```suggestion
pub fn op_add(registers: &mut Registers, instr: u16) {
```
I would make this change here and in every other opcode, it is more accurate to have the name in plural. Also changing the struct's name.

Reply:
Done!